README.md

```diff
@@ -1,2 +1,28 @@
-# kf-sphinx-docs-theme
-💄 Kids First Sphinx Docs Theme
+# Kids First Sphinx Docs Theme
+
+A Kids First styled Sphinx theme based on https://github.com/rtfd/sphinx_rtd_theme
+
+Sphinx is a web documentation generator for Python applications
+
+## Installation
+```
+pip install -e git+https://github.com/kids-first/kf-sphinx-docs-theme.git#egg=kf-sphinx-docs-theme
+```
+
+## Use
+To use the Kids First Sphinx theme simply put the following in your Sphinx configuration file
+(conf.py). See http://www.sphinx-doc.org/en/master/usage/quickstart.html
+for information on how to use Sphinx and what `conf.py` is.
+
+```
+html_theme = 'sphinx_kidsfirst_theme'
+```
```

Comment:
ok, now how to include this in another project in a way that won't immediately fall behind this repo?